builtin/files/userdata/cloud-config-controller

```diff
@@ -1127,6 +1127,11 @@ write_files:
         "${mfdir}/kube-dns-de.yaml"
       {{- end }}
       {{ if .KubeDns.NodeLocalResolver -}}
+      {{ if .KubeDns.dnsmasq.CoreDNSLocal.Enabled -}}
```

Comment:
Casing on this needs to change, we need `DNSMasq`.

Reply:
👍